tests/integration/test_mongodb.py

```diff
@@ -30,8 +30,8 @@
     NoHitsFoundError,
     ResourceNotFoundError,
 )
-from hexkit.providers.mongodb.testutils import mongodb_fixture  # noqa: F401
 from hexkit.providers.mongodb.testutils import MongoDbFixture
```

Comment:
Same idea here, could be imported from `fixtures`.

Reply:
[62c8334](https://github.com/ghga-de/hexkit/pull/59/commits/62c83347c2f2270183439246b4a8bdebafa7039e)